### ISSUE TITLE
Upload from Chrome does not contain file's content

### ISSUE THREAD

**Issue by cod3monk** [vector-im/riot-web#10266]
### Description

Uploading a file from Chrome (Version 76.0.3809.46) on a Mac (10.14.5) produced an invalid file. Further inspection revealed, that the file contained `function stream() { [native code] }` instead of its original content. Uploading through Firefox on the same system and the same file works fine.

Having a look at the development tools and network timeline, the browser already replaced the raw data with the above mentioned string.

The shown file size, however, matches the original file's size, but not of the file downloadable.

### Steps to reproduce

- Enter chat
- Drag file onto chat / or use paperclip icon and select file
- Hit upload on the subsequent dialog
- Download the file
- Compare

Describe how what happens differs from what you expected.

### Version information

- **Platform**: Browser
- **Browser**: Chrome version 76.0.3809.46
- **OS**: macOS 10.14.5
- **URL**: self-hosted riot-web version 1.1.2

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/riot-web/issues/9913
Which should be fixed by https://github.com/matrix-org/matrix-js-sdk/pull/952

Can you confirm your js-sdk version from settings

**Comment by cod3monk** [comment]
Under settings I only see this:
```
Version von matrix-react-sdk: <local>
Version von riot-web: 1.1.2
Version von olm: 3.1.0
```

We will trigger an update and let you know if the problem persists or not.

**Comment by t3chguy** [comment]
Issue was fixed 24 days ago, v1.1.2 is older than that, update your riot

**Comment by cod3monk** [comment]
Update to v1.2.2 fixed the issue. Thanks for the help :)
